app/api/trpc/lambda/[trpc]/route.ts

```
@@ -20,12 +19,23 @@ const handler = async (req: any) => {
   let enrichedUser = null;
 
   if (authUser) {
-    const profile = assertDefined(await getProfile(authUser.id));
-
-    enrichedUser = {
-      email: authUser.email ?? null,
-      ...profile,
-    };
+    const profile = await getProfile(authUser.id);
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)